examples/Language_Translation/English_to_French_Translation/model_handler.py

```diff
@@ -0,0 +1,61 @@
+from ts.torch_handler.base_handler import BaseHandler
+from fairseq.models.transformer import TransformerModel
+import torch
+from pathlib import Path
+import os
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class LanguageTranslationHandler(BaseHandler):
+
+    def __init__(self):
+        self._context = None
+        self.initialized = False
+        self.model = None
+        self.device = None
+
+    def initialize(self, context):
+        self._context = context
+        self.initialized = True
+        self.manifest = context.manifest
+
+        properties = context.system_properties
+        model_dir = properties.get("model_dir")
+
+        self.device = torch.device(
+            "cuda:" + str(properties.get("gpu_id")) if torch.cuda.is_available() else "cpu")
+
+        #  load the model
+        self.model = TransformerModel.from_pretrained(
+            model_dir,
+            checkpoint_file='model.pt',
+            data_name_or_path=model_dir,
+            tokenizer='moses',
+            bpe='subword_nmt'
+        )
+        self.model.to(self.device)
+        self.model.eval()
+
+        self.initialized = True
+
+    def preprocess(self, data):
+        text = data[0].get("data")
```

Comment:
@AshwinChafale It will be good if you can add support for Batch processing as well. Otherwise please add clarification notes that batch processing is not supported

Reply:
Added support for batch processing. I have also updated README.md file content also